samples/new-node_vagrant.yaml

```diff
@@ -0,0 +1,17 @@
+# Use this manifest to announce the new node to the apiserver.
+# You need to fill the needed SSH connection information.
+# CMD: sudo kubectl --kubeconfig /etc/kubernetes/admin.conf create -n "kube-system" -f new-node_vagrant.yaml
+apiVersion: v1
+kind: Node
+metadata:
+  name: node1
+  annotations:
+    metalk8s.scality.com/ssh-user: vagrant 
+    metalk8s.scality.com/ssh-port: 22
+    metalk8s.scality.com/ssh-host: <IP_ADDRESS>
+    metalk8s.scality.com/ssh-key-path: /etc/metalk8s/pki/preshared_key_for_k8s_nodes
+    metalk8s.scality.com/ssh-sudo: "true" 
+spec:
+  taints:
+  - effect: NoSchedule
```

Comment:
@sayf-eddine-scality we also need to add a `labels` entry at the same level as `annotations` otherwise we won't be able to update it during the deployment of the new node

Reply:
Something like this is working:
```
apiVersion: v1
kind: Node
metadata:
  name: node1
  annotations:
    metalk8s.scality.com/ssh-user: vagrant
    metalk8s.scality.com/ssh-port: "22"
    metalk8s.scality.com/ssh-host: 172.21.254.3
    metalk8s.scality.com/ssh-key-path: /etc/metalk8s/pki/preshared_key_for_k8s_nodes
    metalk8s.scality.com/ssh-sudo: "true"
  labels:
spec:
  taints:
  - effect: NoSchedule
    key: node-role.kubernetes.io/master
```

Reply:
arf yes this is the bug found by dudu